fonts.css

```diff
@@ -0,0 +1,44 @@
+/*! Generated by Font Squirrel (https://www.fontsquirrel.com) on October 9, 2020 */
+
+
+
+@font-face {
+    font-family: 'pt_serifbold';
+    src: url('ptserif-bold-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LandingPage/pullRequest?prid=6317324)